object/interp.go

```diff
@@ -0,0 +1,36 @@
+package object
+
+import (
+	"errors"
+)
+
+var commands map[string]Extension
+
+func init() {
+	Init()
+}
+
+func Init() {
+	commands = make(map[string]Extension)
+}
```

Comment:
The init hack should be removed here too

Reply:
if only maps had non useless nil zero values, it's annoying but let me see (again see token.go where init is mandatory)

Reply:
OK

Reply:
I could lazy init in first CreateCommand() I guess but... not sure that's better, it'll be more code

Reply:
fixed it